_posts/2019-03-11-PC-Engines_firmware-v4-9-0-3.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.9.0.3"
+date:   2019-03-11
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.9.0.3
+
+## Key changes
+
+Mainline:
+
+1. **Enabled** CPU **boost** feature in **runtime** configuration. Some
+   platforms seem to have problems with boost thus we decided to make CPU boost
+   runtime configurable.
+2. We have **upgraded** to coreboot **SDK 1.52** in mainline releases.
+3. Configured **pull-ups** on **WLAN_DISABLE#** pins on the mPCIe connectors
+   which could cause issues with certain modems when floating.
```

Comment:
@miczyg1 let's give example of modem model that we know have problems

Reply:
Done